lib/govuk_index/publishing_event_job.rb

```diff
@@ -40,8 +40,10 @@ class MissingExternalUrl < StandardError; end
       worldwide_office_staff_role
     ].freeze
```

Comment:
Do you happen to know if anything terrible will happen if all this stuff moves inside the class definition? As they're in the wrong place in the namespace. Though I appreciate fixing this could break something else.